eng/validation/update-channel.ps1

```diff
@@ -15,34 +15,45 @@ function Get-Headers([string]$accept, [string]$barToken) {
 
 $arcadeSdkPackageName = 'Microsoft.DotNet.Arcade.Sdk'
 $arcadeSdkVersion = $GlobalJson.'msbuild-sdks'.$arcadeSdkPackageName
-$getAssetApiEndpoint = "$maestroEndpoint/api/assets?name=$arcadeSdkPackageName&version=$arcadeSdkVersion&api-version=2018-07-16"
+$getAssetsApiEndpoint = "$maestroEndpoint/api/assets?name=$arcadeSdkPackageName&version=$arcadeSdkVersion&api-version=2018-07-16"
 $headers = Get-Headers 'text/plain' $barToken
 
 # Get the Microsoft.DotNet.Arcade.Sdk with the version $arcadeSdkVersion so we can get the id of the build
-$assets = Invoke-WebRequest -Uri $getAssetApiEndpoint -Headers $headers | ConvertFrom-Json
-
-if ($assets) {
-    # Today, there shouldn't be more that one build linked to a given Arcade SDK version, but if in the future we only bump the version if
-    # changes were done that could be a possibility. Also, since we already validated version $arcadeSdkVersion we move all the builds that
-    # produced it to channel $channelId
-    foreach ($asset in $assets) {
-        try {
-            $buildId = $asset[0].'buildId'
-
-            $postBuildIntoChannelApiEndpoint = "$maestroEndpoint/api/channels/$channelId/builds/$buildId/?name=$arcadeSdkPackageName&version=$arcadeSdkVersion&api-version=2018-07-16"
-            $headers = Get-Headers 'application/json' $barToken
-        
-            Write-Host "POSTing to $postBuildIntoChannelApiEndpoint..."
-            $postResponse = Invoke-WebRequest -Uri $postBuildIntoChannelApiEndpoint -Headers $headers -Method Post
-            Write-Host "Build '$buildId' was successfully added to channel '$channelId'"
-         }
-        catch {
-            Write-Host $_
-            Write-Host $_.ScriptStackTrace
-            exit 1
-        }
-    }
-} else {
+$assets = Invoke-WebRequest -Uri $getAssetsApiEndpoint -Headers $headers | ConvertFrom-Json
+
+if (!$assets) {
     Write-Host "Asset '$arcadeSdkPackageName' with version $arcadeSdkVersion was not found"
     exit 1
-}
\ No newline at end of file
+}
+
+if ($assets.Count > 1) {
```

Comment:
I think this should be -ne 1.  Although I'm not positive, it's possible the assets could just be an empty json array. In that case, we'd pass through the above !$assets and this statement.

Reply:
Just validated and `!$assets` works fine with empty arrays